### ISSUE TITLE
Accessibility : Graph point incorrectly mentions "Image" after circling back from the last chart with Safari Voice Over

### ISSUE THREAD

**Issue by don-siu** [highcharts/highcharts#17171]
Versions:
"highcharts": "^10.0.0",
"highcharts-react-official": "^3.1.0",

1. Using macOS VoiceOver+Safari open the web application with high chart graph
2. Press Ctrl-Opt-right arrow to navigate past the last dot in the chart

Expected Result
Image without alt text is hidden from screen readers.

Actual Result
Image without alt text is focused and announced as 'image'.

HTML:
<rect x="-5.997999906539917" y="147.99935913085938" width="12" height="12" fill="none" rx="3" ry="3" class="highcharts-focus-border" data-z-index="99" stroke="#335cad" stroke-width="2"></rect>

#### Live demo with steps to reproduce
Please see attachment

https://user-images.githubusercontent.com/42818524/162535860-5a5a5184-4e6c-4e8a-89fb-20f6c8dc501d.mov

<img width="544" alt="image" src="https://user-images.githubusercontent.com/42818524/162534310-2b8954c3-a493-43ec-ac06-7f15a3ec881f.png">

#### Product version
NA

#### Affected browser(s)
Safari

**Comment by devtopher** [comment]
Since this issue has been stale for awhile (6+ months) is this something that should be closed?

**Comment by don-siu** [comment]
yes this is still an issue. Can someone fix this?

**Comment by don-siu** [comment]
> Since this issue has been stale for awhile (6+ months) is this something that should be closed?

yes this is still an issue. Can someone fix this? It's been 6+ months

**Comment by devtopher** [comment]
> > Since this issue has been stale for awhile (6+ months) is this something that should be closed?
> 
> yes this is still an issue. Can someone fix this? It's been 6+ months

Thanks for letting us know it's still an issue. It looks like it hasn't been able to be reproduced - do you have a sandbox or demo of a chart which has the question? I have safari and am happy to test it out if you do
